testsuite/replay_verify.py

```diff
@@ -16,6 +16,43 @@
 # This script runs the replay-verify from the root of aptos-core
 # It assumes the aptos-db-tool binary is already built with the release profile
 
+testnet_runner_mapping = {
```

Comment:
can we add a comment on what is this mapping?

Reply:
also why their number is so random? Is this depends on the transaction we need to skip?

Reply:
updated. The number is generated using a script initially linked in the comment as well

Reply:
Should add steps to 1. how we "seal" the latest range when it reaches a similar run time compared with previous ranges (and that the oncall will receive a low pri alert) 2. how we retire the oldest range when according to our policy the backward compatibility is no longer required.

Reply:
(the steps can be easier if we make this an vector instead of a map, and automatically give runners names? (removes the  `strategy.matrix.number` array as well?) -- Looks like in the current way we are coding it, adding a range will require changes in three places.

Reply:
how do we make sure different runners get different ranges without passing in a seq#? Each runner doesn't know about other runner and cannot communicate.

Reply:
The vector items is keyed by the index anyway? I'm a jerk and I didn't read, but in my illusion when you spawn the runners just read the length of the vector, and assign 0..len to the runners

Reply:
I see. Github spawns the runners.  But I guess you mean I can put the ranges in Yaml to coordinator the runners.